staging/src/k8s.io/apiserver/pkg/storage/testing/store_tests.go

```diff
@@ -2759,3 +2759,312 @@ func RunTestListPaging(ctx context.Context, t *testing.T, store storage.Interfac
 		t.Errorf("unexpected items: %#v", names)
 	}
 }
+
+func RunTestNamespaceScopedList(ctx context.Context, t *testing.T, store storage.Interface) {
+	tests := []struct {
+		name               string
+		requestedNamespace string
+		recursive          bool
+		fieldSelector      fields.Selector
+		indexFields        []string
+		gotPods            []example.Pod
```

Comment:
got suggests it's a result, whereas it's input. 

inputPods? existingPods?

Reply:
Done